pkgs/tools/misc/fastfetch/default.nix

```diff
@@ -0,0 +1,69 @@
+{ chafa
+, cmake
+, dbus
+, dconf
+, fetchFromGitHub
+, glib
+, imagemagick_light
+, lib
+, libglvnd
+, libxcb
+, makeWrapper
+, ocl-icd
+, opencl-headers
+, pciutils
+, pkgconfig
+, stdenv
+, vulkan-loader
+, wayland
+, xfce
+, xorg
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fastfetch";
+  version = "1.5.6";
+
+  src = fetchFromGitHub {
+    owner = "LinusDierheimer";
+    repo = pname;
+    rev = version;
+    hash = "sha256-OsK78irfO9r6pSEmlN6mFoKhQZgBqIYO8syj5AHH4IE=";
+  };
+
+  nativeBuildInputs = [ cmake makeWrapper pkgconfig ];
+
+  runtimeDependencies = [
+    chafa
+    dbus
+    dconf
+    glib
+    imagemagick_light
+    libglvnd
+    libxcb
+    ocl-icd
+    pciutils
+    vulkan-loader
+    wayland
+    xfce.xfconf
+    zlib
+  ];
+
+  buildInputs = runtimeDependencies ++ [ xorg.libX11 opencl-headers ];
+
+  LD_LIBRARY_PATH = lib.makeLibraryPath runtimeDependencies;
+
+  postInstall = ''
+    wrapProgram $out/bin/fastfetch --prefix LD_LIBRARY_PATH : "${LD_LIBRARY_PATH}"
+    wrapProgram $out/bin/flashfetch --prefix LD_LIBRARY_PATH : "${LD_LIBRARY_PATH}"
```

Comment:
Must we use LD_LIBRARY_PATH?

Reply:
I tried using autopatchElf to modify the DT_RPATH of the executable but seems like in some cases (DBUS to be specific, atleast the for now), it's not able to find the library via dlopen.

Reply:
I saw on the projects homepage these are optional at runtime even for precompiled binaries, it probably always depends on `LD_LIBRARY_PATH`.